assets/text/effects/genericCampaignPlots_last_imperialPalace_keystone.properties

```diff
@@ -0,0 +1,15 @@
stupid one? Sure, why hold back now?[face:skeptical]/We're basically the same age.[face:dubious]/[face:skeptical]Since when are [i]you[] the youngest?>/<hero2.bookish/coward/goofball/hothead/loner/romantic/60:Like to say there's always time for questions. But is it [i]pertinent?[][face:skeptical]/[face:interested]Nothing weird, is it? Also... <hero.7ysoThan.hero2:do you think of me as [i]old?[] I guess that's fine, I just... hm./do you think of me as [i]older?[] Really?/do you think of me as [i]older?[] Really?>/[face:happy]A little baby one, I guess./<hero.10ysoThan.hero2:I might be a creaky old crookfoot at this point, but/We're the same age, almost! Fine, but/In what way am I older than you? Anyway,> I can still kick your ear. Remember that.[face:skeptical]/You seem to be doing so./<hero.7ysoThan.hero2:Youth's not [i]everything,[] you know?/We're basically the same age./Since when are [i]you[] the youngest?> Anyway, a lot of people find experience attractive. Or wait, what was the question?[face:skeptical]/Guess so. What's up, <hero.pcfn.hero2:<hero.mf:Dad/Mom/Renny>?/<hero.mf:son-of-mine/daughter-mine/kid>?/<hero>?/<hero>?>>/<hero2.coward/goofball/greedy/hothead/loner/romantic/60:Oh, no. [i]Love...[] you think of me as... [i]older?[]/[face:happy]Sure! If I can give you a prettiest-one answer./[face:skeptical]Only if I can give you a prettiest-one answer./[face:joy]<hero.3ysoThan.hero2:You [i]trying[] to get mad? [i]Fine![] Just ask!/That year or two means a [i]lot[] to you, doesn't it? Fine, ask!/[i]Youngest?[] Since when? But sure, ask!>/That's a weird thing to emphasize, love. But sure./[face:happy]Oh, am I suddenly meant to play the alluring older <hero2.mf:guy/lady/character>? The <hero2.whm:brooding knight? Want me to do my gravel voice?/lonesome woods<hero2.mf:man/woman/walker>? Want me to do my whisper-growl?/professor? Want me to be [i]stern?[]>/[face:dubious]Sure. Ask this ancient face your questions, [i]dear[]. Like a magic gargoyle, I'll spit answers.>/<hero2.bookish/coward/goofball/hothead/loner/romantic/60:Like to say there's always time for questions. But is it [i]pertinent?[][face:skeptical]/[face:interested]Nothing weird, is it? Also... <hero.7ysoThan.hero2:do you think of me as [i]old?[] I guess that's fine, I just... hm./do you think of me as [i]older?[] Really?/do you think of me as [i]older?[] Really?>/[face:happy]A little baby one, I guess./<hero.10ysoThan.hero2:I might be a creaky old crookfoot at this point, but/We're the same age, almost! Fine, but/In what way am I older than you? Anyway,> I can still kick your ear. Remember that.[face:skeptical]/You seem to be doing so./<hero.7ysoThan.hero2:Youth's not [i]everything,[] you know?/We're basically the same age./Since when are [i]you[] the youngest?> Anyway, a lot of people find experience attractive. Or wait, what was the question?[face:skeptical]/Guess so. What's up, <hero.pcfn.hero2:<hero.mf:Dad/Mom/Renny>?/<hero.mf:son-of-mine/daughter-mine/kid>?/<hero>?/<hero>?>>>/>>
```

Comment:
I think that in `hero.rfln.hero`, one of those `hero`s should be `hero2`, since a character can't be in a relationship with themselves - looks like that might be a bug in the base game's dialogue too.

Reply:
Yep, it is in the base game dialogue. I'll just send a bug report to the Wildermyth dev's now and have a look at seeing how it's intended to go. Tried changing one of them to hero2 which got the text to display, but it didn't line up quite right even then.